word-addin/src/CyBy/Draw/Word.idr

```diff
@@ -90,9 +90,32 @@ info = log Info
 -- selection of SVGs generated by CyBy-Draw are searched
 -- for. As the RegEx is non-greedy, only the first SVG
 -- occurrence is relevant.
+
+-- As word uses EMU's (English Metric Units) as image sizes,
+-- a conversion from pixels to EUM's had to be done.
+-- 1 Inch = 914400 EMU
+-- 1 Inch = 96 px (as Microsoft uses 96 ppi as standard)
+-- EMU = (914400 / 96) * px = 9525 * px
+-- As EMU should be an Integer , underneath the Nat type is used
+-- here. The difference of the floor rounding is (I think)
+-- negligible.
+record EMU where
+  constructor E
+  value : Nat
+
+-- TODO: Test this
+pxToEMU : (px : Double) -> EMU
+pxToEMU = E . cast . (* 9525)
+
+-- TODO: Test this
+emuAsStr : EMU -> String
+emuAsStr = cast . value
+
 exportImgEmptSel : HasIO io => LogLevel => Selection -> (svg,mol : String) -> io ()
```

Comment:
You were right that the `mol` argument is not needed here. The metadata is already part of the `svg` string here. Sorry, forgot about that.